### ISSUE TITLE
Project state

### ISSUE THREAD

**Issue by nicolas-bastien** [cocur/build-bundle#2]
Hi @florianeckerstorfer, 

This bundle seams to be what I'm looking for but you don't seem to maintain it anymore.
Do you have any plan for it ?
Or maybe you find a better solution ?

Thanks a lot for you response

Nicolas

**Comment by florianeckerstorfer** [comment]
Yes, in the end I never used this project for anything myself and it's dead
